src/sentry/tasks/post_process.py

```diff
@@ -71,26 +70,6 @@ def check_event_already_post_processed(event):
     return not result
 
 
-def check_correct_event_sequencing(event):
```

Comment:
Why is all of this being removed?

Reply:
looks like an issue w/ a rebase - fixing